### PR TITLE
🗃 (prisma): changed nomenclature

### PR DESCRIPTION
**Pull request by KayleePosthumus**

### CODE REVIEW

**Review by Ashleigh-Tuks — APPROVED** [review]

File: prisma/schema.prisma [hunk 1]
```diff
@@ -15,87 +15,92 @@ datasource db {
...
+  created      DateTime
+  suspended    Boolean
+  validated    Boolean
+
+  company User? @relation(name: "company_id", fields: [companyId], references: [id])
+
+  User             User[]            @relation("company_id")
+  userScout        UserScouted[]     @relation("userScout")
+  userScouted      UserScouted[]     @relation("userScouted")
+  UserRole         UserRole[]
```
Comment (Ruben-Brits) [inline prisma/schema.prisma:35]: should these not all start with lower case to follow the convention of the rest of the models? :)

Comment (KayleePosthumus) [reply prisma/schema.prisma:35]: @Ruben-Brits 

Please see https://github.com/COS-301/graduates/pull/213

Comment (Ruben-Brits) [reply prisma/schema.prisma:35]: I saw yes, just wanted to know if the field names like line 35 should not be starting with lower case? UserRole -> userRole and so on for some of the fields.

Comment (KayleePosthumus) [reply prisma/schema.prisma:35]: those are prisma auto generated elements they do not reflect anywhere but I will impose some standard on them in the future.

**Pull request merged**